src/bokeh/core/_templates/autoload_nb_js.js

```diff
@@ -157,24 +157,46 @@
      "</code>\n"+
      "</div>"}};
 
-  function display_loaded() {
+  function display_loaded(error = null) {
     const el = document.getElementById({{ elementid|tojson }});
     if (el != null) {
-      el.textContent = "BokehJS is loading...";
-    }
-    if (root.Bokeh !== undefined) {
-      if (el != null) {
-        el.textContent = "BokehJS " + root.Bokeh.version + " successfully loaded.";
+      const text = (() => {
+        if (typeof root.Bokeh === "undefined") {
+          if (error == null) {
+            return "BokehJS is loading ...";
+          } else {
+            return "BokehJS failed to load.";
+          }
+        } else {
+          return `BokehJS ${root.Bokeh.version} ${error == null ? "successfully" : "partially"} loaded.`
```

Comment:
I think "partially loaded" will be confusing, does that mean some things work? Nothing will work? "It depends"? It's also probably easy to miss since it has the identical phrase structure. I'd suggest something like 
> BokehJS encountered errors while loading and may not function as expected

Possibly with some kind of bold or highlight. 

Also, since the errors might be very long, then *ideally*  think they'd be in a collapsed div with a "click to see console output" activator